src/Template/SentencesLists/show_angular.ctp

```diff
@@ -186,37 +187,37 @@ $this->set('title_for_layout', $this->Pages->formatTitle($listName));
 
     <md-content ng-cloak>
     <?php
-    if ($permissions['canAddSentences'] && count($sentencesInList) == 0) {
+    if (count($sentencesInList) == 0) {
         ?>
         <div class="no-sentence-info" ng-if="!vm.showForm && vm.sentences.length === 0">
-            <p><?= __('This list is empty.') ?></p>
-            <div class="hint">
+            <p>
                 <?php
-                echo format(
-                    __(
-                        'You can create new sentences directly in this list by clicking on the '.
-                        '{addSentenceButton} icon in the header of this section.', true
-                    ),
-                    ['addSentenceButton' => '<md-icon>add</md-icon>']
-                );
+                    echo format(
+                        __(
+                            $filterLanguage=='und'?'This list is empty.':'This list does not contain any sentences in {language}'
```

Comment:
Putting the `'und'` inside the `__()` makes xgettext think that `'und'` is a message that needs to be translated. Better have two separate calls to `__()` in the two branches of the ternary operator.

Also, I'd add a period after `{language}`, ending the sentence without a period looks weird to me.